src/main/java/duke/gui/MainWindow.java

```diff
@@ -0,0 +1,64 @@
+package duke.gui;
+
+import duke.main.Duke;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
```

Comment:
Maybe consider importing duke packages after importing javafx packages and leave a line between for coding styles!